go/vt/discovery/replicationlag.go

```diff
@@ -19,35 +19,47 @@ package discovery
 import (
 	"flag"
 	"fmt"
-	"math"
+	"sort"
 	"time"
 )
 
 var (
 	// lowReplicationLag defines the duration that replication lag is low enough that the VTTablet is considered healthy.
 	lowReplicationLag            = flag.Duration("discovery_low_replication_lag", 30*time.Second, "the replication lag that is considered low enough to be healthy")
-	highReplicationLagMinServing = flag.Duration("discovery_high_replication_lag_minimum_serving", 2*time.Hour, "the replication lag that is considered too high when selecting miminum 2 vttablets for serving")
+	highReplicationLagMinServing = flag.Duration("discovery_high_replication_lag_minimum_serving", 2*time.Hour, "the replication lag that is considered too high when selecting the minimum num vttablets for serving")
+	minNumTablets                = flag.Int("min_number_serving_vttablets", 2, "the minimum number of vttablets that will be continue to be used even with low replication lag")
 )
 
+// TestSetMinNumTablets is a test helper function, if this is used by a production code path, something is wrong.
+func TestSetMinNumTablets(newMin int) {
```

Comment:
This can be moved to replicationlag_test.go, and renamed to testSetMin...

Reply:
Wow that works, updated. Thanks @sougou!